WalletWasabi/WabiSabi/Client/AliceClient.cs

```diff
@@ -167,7 +171,7 @@ private async Task<bool> TryConfirmConnectionAsync(IEnumerable<long> amountsToRe
 
 		if (effectiveAmount <= Money.Zero)
 		{
-			throw new InvalidOperationException($"Round({ RoundId }), Alice({ AliceId}): Adding this input is uneconomical.");
+			throw new InvalidOperationException($"Round({RoundId}), Alice({AliceId}): Adding this input is uneconomical.");
```

Comment:
All the `if` statement makes no sense because it can never enter into it. An uneconomical input is rejected immediately and can never reach the input confirmation phase.

Reply:
ACK, removed it.